guide/miscellaneous/cache-customization.md

```diff
@@ -0,0 +1,93 @@
+# Cache customization
+
+Sometimes, we would like to be able to customize discord.js's caching behavior in order to reduce memory usage.
```

Comment:
When explaining parts of a guide, it's recommended to use "you" instead of "we" in most situations.

https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md